doc/usage.rst

```diff
@@ -124,10 +125,17 @@ options::
                         baseline_python, not changed_python.
   --append FILENAME     Add runs to an existing file, or create it if
                         it doesn't exist
+  --min-time MIN_TIME   Minimum duration in seconds of a single value, used
+                        to calibrate the number of loops
+  --same-loops SAME_LOOPS
+                        Use the same number of loops as a previous run
+                        (i.e., don't recalibrate). Should be a path to a
+                        .json file from a previous run.
+  --timeout TIMEOUT     Timeout for a benchmark run (default: disabled)
```

Comment:
What's the unit here?  Seconds?

Reply:
yes, they are seconds. I think I should be explicit. I'll address it tomorrow.

Reply:
Added.